cypress/integration/seededFlows/adminFlows/config/emailServerSettingsSection.spec.js

```diff
@@ -0,0 +1,57 @@
+describe('Email Server Settings Section', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginUser(user);
+    });
+  });
+
+  describe('email server settings', () => {
+    it('updates the smtp fields', () => {
+      cy.get('@user').then(() => {
+        cy.visit('/admin/customization/config');
+        cy.findByTestId('emailServerSettings').as('emailServerSettings');
+
+        cy.get('@emailServerSettings')
+          .findByText('Email Server Settings (SMTP)')
+          .click();
+
+        cy.get('@emailServerSettings')
+          .get('#settings_smtp_user_name')
```

Comment:
can we target by the label text - e.g. `cy.findByLabelText('label for this input')`? Same question for the other input fields below - it makes our tests a bit more robust / closer to how a user experiences the page 🙂

Reply:
Thanks for picking this up @aitchiss - I knew this, not sure why I used ID's.  It also helped bring to light that the for labels weren't matching, which I fixed.